syntax = "proto3";

package api.user;

message UserSignUpRequest {
  string username = 1;
  string password = 2;
}

message UserLoginRequest {
  string username = 1;
  string password = 2;
}

message UserCommonResponse {
  string username = 1;
  string token = 2;
}
